pages/account/manage/link/[[...data]].js

```diff
@@ -211,6 +211,7 @@ export default function ManageLink({ BASE_URL, username, link, groups }) {
                       required
                       minLength="2"
                       maxLength="256"
+                      title="This link cannot be edited because it is predefined by the system."
```

Comment:
The url is not predefined by the system, the user enters their url for the link, but they cannot change it later because it would mess up the stats for the url

I think better wording would be? 

```suggestion
                      title="The URL cannot be edited as this would effect your stats. You can disable this link and create another link."
```

@amandamartin-dev what do you think?

Reply:
Agree with the update now that the issue has been clarified